docs/targets.md

```diff
@@ -14,6 +14,7 @@ cloudposse-atlantis       0.9.0.3    Terraform For Teams, enhanced by Cloud Poss
 codefresh                 0.36.0     Codefresh CLI
 ctop                      0.7.2      Top-like interface for container metrics
 direnv                    2.20.0     Unclutter your .profile
+duffle                    0.3.5      CNAB installer
```

Comment:
Did you add this manually? This is odd. It should be sorted automatically when you run `make readme`

Reply:
I did not, I'll update and push again though.